tests/RefundTest.php

```diff
@@ -8,28 +8,42 @@ class RefundTest extends TestCase
     public function testCreate()
     {
         $charge = self::createTestCharge();
-        $ref = $charge->refunds->create(array('amount' => 100));
-        $this->assertSame(100, $ref->amount);
-        $this->assertSame($charge->id, $ref->charge);
+        $refund = Refund::create(array('amount' => 100, 'charge' => $charge->id));
```

Comment:
Can we leave a few tests in place that the old refund creation call style still works?